data/filters/youtube-cleanup.yaml

```diff
@@ -19,9 +27,72 @@ template: |
   www.youtube.com###info > * [id^="top-level-buttons"] [class*="button-renderer"] [id="text"]:not(:has-text(/\d/))
   www.youtube.com##.ytd-watch-metadata > .top-level-buttons #text:not(:has-text(/[\d]/))
   {{/if}}
+  {{#if remove-comment-section}}
+  www.youtube.com###comments #contents:remove()
+  {{/if}}
+  {{#if hashtags}}
+  www.youtube.com###super-title
+  www.youtube.com##.super-title
+  {{/if}}
 tests:
   - params: {}
     output: ""
+  - params:
+      hashtags: true
+      remove-comment-section: true
+      remove-text-after-buttons-below-video: true
+    output: |
+      www.youtube.com###super-title
+      www.youtube.com##.super-title
+      www.youtube.com###comments #contents:remove()
+      www.youtube.com###info > * [id^="top-level-buttons"] [class*="button-renderer"] [id="text"]:not(:has-text(/\d/))
+      www.youtube.com##.ytd-watch-metadata > .top-level-buttons #text:not(:has-text(/[\d]/))
+  - params:
+      hashtags: true
+      remove-comment-section: true
+      channel-clarification: true
+    output: |
+      www.youtube.com###super-title
+      www.youtube.com##.super-title
+      www.youtube.com###comments #contents:remove()
+      www.youtube.com###clarify-box
+      www.youtube.com##ytd-shorts .disclaimer-container:upward(#info-panel)
+  - params:
+      hashtags: true
+      remove-text-after-buttons-below-video: true
+      channel-clarification: true
+    output: |
+      www.youtube.com###super-title
+      www.youtube.com##.super-title
+      www.youtube.com###info > * [id^="top-level-buttons"] [class*="button-renderer"] [id="text"]:not(:has-text(/\d/))
+      www.youtube.com##.ytd-watch-metadata > .top-level-buttons #text:not(:has-text(/[\d]/))
+      www.youtube.com###clarify-box
+      www.youtube.com##ytd-shorts .disclaimer-container:upward(#info-panel)
+  - params:
+      remove-comment-section: true
+      remove-text-after-buttons-below-video: true
+      channel-clarification: true
+    output: |
+      www.youtube.com###comments #contents:remove()
+      www.youtube.com###info > * [id^="top-level-buttons"] [class*="button-renderer"] [id="text"]:not(:has-text(/\d/))
+      www.youtube.com##.ytd-watch-metadata > .top-level-buttons #text:not(:has-text(/[\d]/))
+      www.youtube.com###clarify-box
+      www.youtube.com##ytd-shorts .disclaimer-container:upward(#info-panel)
+  - params:
+      hashtags: true
+      channel-clarification: true
+    output: |
+      www.youtube.com###clarify-box
+      www.youtube.com##ytd-shorts .disclaimer-container:upward(#info-panel)
+      www.youtube.com###super-title
+      www.youtube.com##.super-title
+  - params:
+      remove-comment-section: true
+      channel-clarification: true
+    output: |
+      www.youtube.com###clarify-box
+      www.youtube.com##ytd-shorts .disclaimer-container:upward(#info-panel)
+      www.youtube.com###comments #contents:remove()
```

Comment:
Thanks for being thorough, but some of these tests fail because the order is not matching (see [the CI logs](https://github.com/xvello/letsblockit/runs/6513974209?check_suite_focus=true#step:6:115)).
As these options are independent, we don't need to test all the combinations of options. The last two you added are enough, all of these can be removed for simplicity.

Reply:
You only have to keep 

```
  - params:
      remove-video-hashtags: true
      channel-clarification: true
    output: |
      www.youtube.com###clarify-box
      www.youtube.com##ytd-shorts .disclaimer-container:upward(#info-panel)
      www.youtube.com###super-title
      www.youtube.com##.super-title
  - params:
      remove-comment-section: true
      channel-clarification: true
    output: |
      www.youtube.com###clarify-box
      www.youtube.com##ytd-shorts .disclaimer-container:upward(#info-panel)
      www.youtube.com###comments #contents:remove()
```

I suppose

Reply:
Actually, I'd only keep the test cases with a single option on (and the first one, with no option on), as we also want to ensure we don't cross wires (remove-video-hashtags actually enabling the rules intended for channel-clarification, and vice-versa)

Reply:
See https://github.com/xvello/letsblockit/pull/153/commits/d55073975515b68f21a962c3c5c76810a5f0fa9e